.github/workflows/integration.yml

```diff
@@ -2,14 +2,14 @@ name: Integration Tests
 
 on:
   pull_request:
-    types: [synchronize, opened, reopened, ready_for_review]
+    types: [synchronize, opened, reopened, ready_for_review, labeled]
   push:
     branches:
       - main
 
 jobs:
   consecutiveness:
-    if: github.event.pull_request.draft == false
+    if: github.event_name == 'push' || contains(github.event.pull_request.labels.*.name, 'trigger-integration-tests')
```

Comment:
I don't understand that this is an `OR`. If it's push will we run the workflow in that way. I think we have the triggers with the `labeled` or `synchronised` and then just need to have:
```
 if: contains(github.event.pull_request.labels.*.name, 'trigger-integration-tests')
```

Reply:
This is needed because that condition is also evaluated at `push` and there is no pull_request event on push

Reply:
> This is needed because that condition is also evaluated at `push` and there is no pull_request event on push

Exactly, we need to leave this if we want the integration tests to run when the PR is merged to main.

Reply:
AAAAAAAAAAAH!!!! My bad :pray: Thanks Pinkie for catching it!